evaluation/evaluation.py

```
@@ -197,6 +200,96 @@ async def aquery(self, *args, **kwargs):
+            return resp
+        new_args = list(args)
+        if new_args:
+            if isinstance(new_args[0], str):
+                new_args[0] = rephrased
+            else:
+                qb = new_args[0]
+                if hasattr(qb, "query_str"):
+                    try:
+                        qb.query_str = rephrased
+                    except Exception:
+                        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)